### PR TITLE
Move Implementation file to Markdown

### PR DESCRIPTION
**Pull request by pmatos**

### CODE REVIEW

File: IMPLEMENTATION.md [hunk 1]
```diff
@@ -32,14 +30,13 @@ additional tag word to further refine the pointer-tag type.
 
 See also:
 
-  Don't Stop the BiBOP: Flexible and Efficient Storage Management for
-  Dynamically Typed Languages.
-  R. Kent Dybvig, David Eby, and Carl Bruggeman.
-  Indiana University TR #400, 1994.
+> *Don't Stop the BiBOP: Flexible and Efficient Storage Management for Dynamically Typed Languages.* by R. Kent Dybvig, David Eby, and Carl Bruggeman, Indiana University TR #400, 1994.
```
Comment (samth) [inline IMPLEMENTATION.md:33]: Link here: http://www.cs.indiana.edu/ftp/techreports/TR400.pdf

File: IMPLEMENTATION.md [hunk 2]
```diff
@@ -98,17 +98,12 @@ and continuation operations are handled as needed at the boundaries.
...
 See also:
  
- Representing Control in the Presence of First-Class Continuations.
- Robert Hieb, R. Kent Dybvig, and Carl Bruggeman.
- Programming Language Design and Implementation, 1990.
-
- Compiler and Runtime Support for Continuation Marks.
- Matthew Flatt and R. Kent Dybvig.
- Programming Language Design and Implementation, 2020.
+> *Representing Control in the Presence of First-Class Continuations* by Robert Hieb, R. Kent Dybvig, and Carl Bruggeman, Programming Language Design and Implementation, 1990.
```
Comment (samth) [inline IMPLEMENTATION.md:101]: https://legacy.cs.indiana.edu/~dyb/pubs/stack.pdf

File: IMPLEMENTATION.md [hunk 3]
```diff
@@ -98,17 +98,12 @@ and continuation operations are handled as needed at the boundaries.
...
+> *Compiler and Runtime Support for Continuation Marks* by Matthew Flatt and R. Kent Dybvig, Programming Language Design and Implementation, 2020.
```
Comment (samth) [inline IMPLEMENTATION.md:102]: https://www.cs.utah.edu/plt/publications/pldi20-fd.pdf

**Comment by samth** [comment]
Ah, I started adding comments too late :)

**Comment by mflatt** [comment]
I merged quickly because I'm going to revise now, so further comments are welcome.

**Pull request merged**